sailthru/sailthru_content/services/tests/catalog_api_test_mixins.py

```diff
@@ -45,3 +53,12 @@ def prepare_get_programs(self):
             content_type='application/json',
             match_querystring=False
         )
+
+    def prepare_searchable_courses(self):
+        responses.add(
+            responses.GET, self.api_url_root + '/search/all/facets/',
```

Comment:
Can you explain to me why the dictionary approach is needed at all?  It looks like we do the same for programs, do you have any idea why?